libkbfs/reporter_kbpki.go

```diff
@@ -83,6 +83,7 @@ type ReporterKBPKI struct {
 	notifySyncBuffer chan *keybase1.FSPathSyncStatus
 	suppressCh       chan time.Duration
 	canceler         func()
+	ctx              context.Context
```

Comment:
I know @jzila suggested this, but I disagree: you're really not supposed to save a context in a struct.  Can you get by with just saving the `Done()` channel instead?  The `ctx` should continue to be passed around explicitly.

Reply:
I think so:

> Successive calls to Done return the same value.

I don't have a strong opinion on any of these. So if that's a solution both of you agree on, I'll make the change. Please let me know. @jzila

Reply:
Strib is correct about the general case, but this isn't a real context here. It's a cancelation proxy.

Reply:
So why not just have the cancellation piece of it, instead of the whole context?

Also, it's being used for more than that, it's passed into all the `CDebugf` statements.

Reply:
But sure, we can save the `Done()` channel instead.